xml_inscriptions/metadata_only/RI.Prov.RISD.MA.Etr.31.364.xml

```diff
@@ -19,7 +19,8 @@
             <publicationStmt>
                
                 <authority>Brown University</authority> 
-                <idno type="USEpigraphy" xml:id="RI.Prov.RISD.MA.Etr.31.364">RI.Prov.RISD.MA.Etr.31.364</idno>
+                <idno type="USEpigraphy" xml:id="RI.Prov.RISD.MA.ETR.31.364"
```

Comment:
You need to change the filename too, if you're going to change the `@xml:id`; otherwise the system is not going to be able to find the inscription.